app/models/response.rb

```diff
@@ -217,4 +217,83 @@ def self.get_volume_of_review_comments(assignment_id, reviewer_id)
     avg_vol_in_round_3 = (Lingua::EN::Readability.new(comments_in_round_3).num_words / (counter_in_round_3.zero? ? 1 : counter_in_round_3)).round(0)
     [overall_avg_vol, avg_vol_in_round_1, avg_vol_in_round_2, avg_vol_in_round_3]
   end
+  def notify_on_difference(new_pct, avg_pct, limit,response_id)
+    mapping = self.map
+    instructor = mapping.assignment.instructor
+    team_id = mapping.reviewee_id
+    teams_users = TeamsUser.where(team_id: team_id)
+    user_id=nil
+    teams_users.each do |team_user|
+      user_id=team_user.user_id;
+    end
+    current_user = User.find_by(id: user_id)
+    @student_tasks = StudentTask.from_user current_user
+    @student_tasks.each do |student_task|
+      if student_task.assignment.id == mapping.assignment.id
+        team_id = student_task.participant.id
+      end
+    end
+    Mailer.generic_message(
+        {:to => instructor.email,
+         :subject => "Expertiza Notification: A review score is outside the acceptable range",
+         :body => {
+             :first_name => ApplicationHelper::get_user_first_name(instructor),
+             :reviewer_name => mapping.reviewer.fullname,
+             :type => "review",
+             :reviewee_name => mapping.reviewee.fullname,
+             :limit => limit,
+             :new_pct => new_pct,
+             :avg_pct => avg_pct,
+             :types => "reviews",
+             :performer => "reviewer",
+             :assignment => mapping.assignment,
+             :partial_name => 'limit_notify',
+             :conflicting_response_url => 'https://expertiza.ncsu.edu/response/view?id='+response_id.to_s,  #'https://expertiza.ncsu.edu/response/view?id='
+             :summary_url => 'https://expertiza.ncsu.edu/grades/view_team?id='+team_id.to_s #mapping.reviewee_id.to_s
+         }
+        }
+    ).deliver_now
+  end
+  def self.compare_scores(new_response, questionnaire)
+    map_class = new_response.map.class
+    existing_responses = map_class.get_assessments_for(new_response.map.reviewee)
+    total, count = get_total_scores_for_prev_reviews(existing_responses, new_response)
+    if count > 0
```

Comment:
I don't think this is a good idea, this means what if this reviewee received more than 1 review from other reviewer, this will be triggered and 'notify_instructor' will be called.